### ISSUE TITLE
Pixelate not responding after loading Bitmap in ImageView

### ISSUE THREAD

**Issue by taimur97** [DanielMartinus/Pixelate#2]
I can't find anything wrong in my integration I have also attached my project files.
It's just a simple demo project please check if there is anything wrong with my integration
//Main MainActivity

```Java
package com.example.project;


import android.app.Activity;
import android.content.Intent;
import android.graphics.Bitmap;
import android.net.Uri;
import android.os.Bundle;
import android.provider.MediaStore;
import android.view.View;
import android.widget.ImageView;
import android.widget.SeekBar;
import android.widget.TextView;

import com.melnykov.fab.FloatingActionButton;

import java.io.IOException;

import nl.dionsegijn.pixelate.Pixelate;

public class MainActivity extends Activity{

    FloatingActionButton fab;
    Pixelate imageView;
    TextView tv;
    SeekBar seekBar;
    private static final int PICK_IMAGE_REQUEST  = 700;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity_layout);
        fab = (FloatingActionButton) findViewById(R.id.fab);
        imageView = (Pixelate) findViewById(R.id.MainImageView);
        tv = (TextView) findViewById(R.id.tvMain);
        seekBar = (SeekBar) findViewById(R.id.seekBar);

        try {
            fab.bringToFront();
        } catch (Exception e) {
            e.printStackTrace();
        }
        fab.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent();
// Show only images, no videos or anything else
                intent.setType("image/*");
                intent.setAction(Intent.ACTION_GET_CONTENT);
// Always show the chooser (if there are multiple options available)
                startActivityForResult(Intent.createChooser(intent, "Select Picture"), PICK_IMAGE_REQUEST);
            }
        });
        seekBar.setMax(60);
        seekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
            @Override
            public void onProgressChanged(SeekBar seekBar, int i, boolean b) {
                imageView.pixelate(i);
            }

            @Override
            public void onStartTrackingTouch(SeekBar seekBar) {

            }

            @Override
            public void onStopTrackingTouch(SeekBar seekBar) {

            }
        });
    }

    @Override
    protected void onResume() {
        super.onResume();

    }

    @Override
    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);

        if (requestCode == PICK_IMAGE_REQUEST && resultCode == RESULT_OK && data != null && data.getData() != null) {

            Uri uri = data.getData();
            try {
                Bitmap bitmap = MediaStore.Images.Media.getBitmap(getContentResolver(), uri);
                imageView.setImageBitmap(bitmap);
                imageView.pixelate(20);
                fab.setVisibility(View.GONE);
                tv.setVisibility(View.GONE);
                seekBar.setVisibility(View.VISIBLE);
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
    }

}
```

```XML
//My Layout File
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:fab="http://schemas.android.com/apk/res-auto"
    android:orientation="vertical"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <nl.dionsegijn.pixelate.Pixelate
        android:layout_width="fill_parent"
        android:layout_height="fill_parent"
        android:id="@+id/MainImageView"
        android:layout_centerInParent="true"
        android:scaleType="centerCrop"
        />
    <TextView
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:text="Click Add Button to Select Image"
        android:id="@+id/tvMain"
        android:textSize="25sp"
        android:layout_centerVertical="true"
        android:layout_centerHorizontal="true" />

    <com.melnykov.fab.FloatingActionButton
        android:id="@+id/fab"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="bottom|right"
        android:src="@drawable/add"
        fab:fab_colorNormal="@color/colorPrimary"
        fab:fab_colorPressed="@color/colorPrimaryDark"
        fab:fab_colorRipple="@color/colorAccent"
        android:layout_marginBottom="28dp"
        android:layout_alignParentBottom="true"
        android:layout_alignRight="@+id/tvMain"
        android:layout_alignEnd="@+id/tvMain" />

    <SeekBar
        android:layout_width="fill_parent"
        android:layout_height="wrap_content"
        android:visibility="gone"
        android:id="@+id/seekBar"
        android:layout_alignParentBottom="true"
        android:layout_alignParentLeft="true"
        android:layout_alignParentStart="true"
        android:layout_marginBottom="27dp"
        android:layout_alignParentRight="true"
        android:layout_alignParentEnd="true" />
</RelativeLayout>
```

**Comment by DanielMartinus** [comment]
@taimur97 Hi andrew, thanks for creating this issue. I've created a test project and it worked fine during a simple test with the scenario:
- Creating an Pixelate imageview
- Set image source in XML
- Pixelate the imageview

Though your scenario is setting the image bitmap after creating the Pixelate imageview what should work perfectly fine. So to answer your question, your integration is correct and should work. I've been able to reproduce your situation and it indeed didn't pixelate the Imageview. I have my assumptions why it didn't work and will fix it this evening (GMT+1).
